internal/provider/resource_docker_volume.go

```diff
@@ -129,6 +135,11 @@ func resourceDockerVolumeRead(ctx context.Context, d *schema.ResourceData, meta
 }
 
 func resourceDockerVolumeDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+
+	if prevent_destroy := d.Get("prevent_destroy").(bool); prevent_destroy {
```

Comment:
```suggestion
	if d.Get("prevent_destroy").(bool) {
```

In Go, you should use `camelCase` rather than `snake_case`.

Reply:
I updated and squashed into my original commit